mono/metadata/domain-internals.h

```diff
@@ -608,7 +608,12 @@ MonoJitInfo* mono_jit_info_table_find_internal (MonoDomain *domain, char *addr,
 void mono_enable_debug_domain_unload (gboolean enable);
 
 MonoReflectionAssembly *
-mono_domain_try_type_resolve_checked (MonoDomain *domain, char *name, MonoObject *tb, MonoError *error);
+mono_domain_try_type_resolve_name (MonoDomain *domain, const char *name, MonoError *error);
+
+struct _MonoReflectionTypeBuilder;
```

Comment:
Remove that

Reply:
The struct? I believe it is needed.

Reply:
The line break on mono_class_get_method_from_name_checked was to try to fit in 80 columns